frontend/src/App.tsx

```diff
@@ -20,6 +20,9 @@ const Routes: React.FunctionComponent = () => {
       <Route exact path="/sightings">
         <BrowseSightings />
       </Route>
+      <Route path="/sightings/:speciesId">
```

Comment:
Should mimic the backend and specify it's by species ID, in case we also do location later
```suggestion
      <Route path="/sightings/species/:speciesId">
```

Reply:
done, thanks